adal/devApps/XFormsApp/SecondPage.cs

```diff
@@ -202,7 +208,54 @@ void LogCallback(LogLevel level, string message, bool containsPii)
                     this.result.Text += "Logs : " + DrainLogs();
                 });
             }
+        }
 
+        private async void AcquireTokenWithBrokerButton_Clicked(object sender, EventArgs e)
```

Comment:
I believe Travis enabled some static analysis related to async. One of the rules is that all Async methods should be suffixed with Async.

Reply:
Perspective:
The rule is not meaningful for top level event handlers and can be ignored imo. This is the entry point for the async call. However either ignore and do not apply any async stuff to it or leave it with everything that the async rules says.

Reply:
Yes, agreed, especially as VS generates those even handlers from the designer. Irrespective of the solution chosen we should try not to introduce new warnings into the build. 

The number of warnings has dropped a lot in the past few weeks and once it reaches 0 we can enable "treat warnings as errors" and start looking at more static analysis.

Reply:
I agree. I will leave this as-is. Thanks.